### DISCUSSION TITLE
ADC - Soil Evaluation Curve - Coefficent generation undocumented?!

### DISCUSSION THREAD

**Discussion by mrred2k** [rbaron/b-parasite#195] [Q&A]
As I am testing coatings, and wanted to calibrate the devices based on multiple coatings etc, I couldnt find any documentation on how to produce coeffitions from adc raw soul measurements. 
I checked the functions, which are producting the estimated "percentages" but I was not able to change the coeffs in the device tree. 

The 3 coefficients are produced somewhere else but I dont know where. 

@rbaron Maybe you can help here a little bit. I am more than willing to support you with the docs in some PR.

**Comment by rbaron** [comment]
@mrred2k indeed the calibration is not very well documented. I tried explaining what I did in https://github.com/rbaron/b-parasite/issues/79#issuecomment-1637418626 and  https://github.com/rbaron/b-parasite/issues/79#issuecomment-1650278703, but it would be ideal to write a better process for it -- or even better, automate it.
